cypress/integration/spec-theme.ts

```diff
@@ -0,0 +1,16 @@
+describe('The theme', () => {
+  before(() => {
+    cy.roomEnter(true)
+  })
+  
+  after( () => {
+    cy.logout()
+  })
+  
+  // @setBackground
+  it('should set the background to the room', () => {
+    cy.fixture('theme').then(theme => {
+      cy.get('.room-content').should('have.css', 'background-color', theme.generatedBackgroundColor)
```

Comment:
Pour la selection des attributs il faudrait toujours les sélectionner grâce à un attribut `data-testid`, si ça manque sur cet élément hésite pas à la rajouter

Reply:
Yes you're right. I will add the changes in a dedicated branch.